src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -286,12 +236,9 @@ public function settingsSummary() {
         case 'metadatadisplayentity':
           $entity = NULL;
           if ($this->getSetting('metadatadisplayentity_source')) {
-            $entity = $this->entityTypeManager->getStorage(
-              'metadatadisplay_entity'
-            )->load($this->getSetting('metadatadisplayentity_source'));
+            $entity = $this->entityTypeManager->getStorage('metadatadisplay_entity')->load($this->getSetting('metadatadisplayentity_source'));
```

Comment:
For readability, I prefer methods for long lines (over 80 chars) to be split amongst multiple lines. But that said, I'm also fine. MY IDE (even when using DCS and Symfony ones) breaks them automatically. So if you see me breaking them and you are not into that let me know!